accounts/abi/reflect.go

```diff
@@ -85,3 +85,12 @@ func set(dst, src reflect.Value, output Argument) error {
 	}
 	return nil
 }
+
+// requireAssignable verifies that `dest` and `src` types are compatible
+// and it's possible to safely assign `src` value to `dest`.
```

Comment:
This does not appear to be what it's actually doing.

Reply:
Right, there is better way to document it. Howe about:

// requireAssignable verifies that `dest` type is assignable.

Reply:
That works better.

Reply:
changed to:
```
// requireAssignable assures that `dest` is a pointer and it's not an interface.
```